### ISSUE TITLE
Accordion - closes when accordion item is clicked

### ISSUE THREAD

**Issue by szilvihorvath** [Capgemini/dcx-react-library#660]
**Describe the bug**
When an accordion item is expanded and the item's details is clicked, the accordion item closes.

**To Reproduce**
1. Open an accordion section
2. Click on the accordion details (not the accordion title)
3. The accorion section will close

**Expected behavior**
The accordion item should only close when the accordion title is clicked.

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: chrome
 - Version 125.0.6422.61

**Additional context**
The Accordion has an onClick function that is passed down to each accordion item including the AccordionDetails, so when you click on any part of an accordion item, it closes the item.

Please follow these steps to create your branch:
```bash
git checkout release/1.1.0
git pull
git checkout -b 'fix/accordion-click-event'
```

**Comment by alexsteele95** [comment]
Now looking into this!

**Comment by alexsteele95** [comment]
Should be fixed by: [https://github.com/Capgemini/dcx-react-library/pull/665](https://github.com/Capgemini/dcx-react-library/pull/665) 
cc: @szilvihorvath @daniele-zurico

**Comment by daniele-zurico** [comment]
Closed by #665
